generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p><code>config_file</code> -&gt; <code><code>config_file</code></code></p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
The visualization is not good above. I mean adding ` to the config_file.
```
`config_file`
```

Reply:
Thanks, done